terraform/environments/panda-cyber-appsec-lab/platform_backend.tf

```diff
@@ -9,6 +9,6 @@ terraform {
     encrypt              = true
     key                  = "terraform.tfstate"
     region               = "eu-west-2"
-    workspace_key_prefix = "environments/members/panda-cyber-appsec-lab" # This will store the object as environments/members/panda-cyber-appsec-lab/${workspace}/terraform.tfstate
+    workspace_key_prefix = "environments/members/$application_name" # This will store the object as environments/members/$application_name/${workspace}/terraform.tfstate
```

Comment:
This will need to accurately reflect the customer account name

Reply:
I will amend this and repost

Reply:
Amended and pushed up